mozilla/solitude
# upgrade django

* upgrade django and django-nose
* remove IN_TEST_SUITE
* move to ATOMIC_REQUESTS
* add in rollback that doesn't barf in tests
* add tests to braintree
* ensure that one wierd test case in Bango status still works
